src/App.jsx

```diff
@@ -14,57 +16,63 @@ export const goods = [
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+export const App = () => {
+  const [goodSelected, setGoodSelected] = useState('Jam');
 
-    <h1 className="title is-flex is-align-items-center">
-      Jam is selected
-      <button data-cy="ClearButton" type="button" className="delete ml-3" />
-    </h1>
+  return (
+    <main className="section container">
+      {goodSelected === null ? (
+        <h1 className="title is-flex is-align-items-center">
+          No goods selected
+        </h1>
+      ) : (
+        <h1 className="title is-flex is-align-items-center">
+          {goodSelected} is selected
+          <button
+            data-cy="ClearButton"
+            type="button"
+            className="delete ml-3"
+            onClick={() => setGoodSelected(null)}
+          />
+        </h1>
+      )}
+      <table className="table">
+        <tbody>
+          {goods.map((item, index) => {
+            return (
+              <tr
+                data-cy="Good"
+                // eslint-disable-next-line react/no-array-index-key
+                key={index}
```

Comment:
You are using index as a key which is not recommended according to the checklist. This is a bad practice because the key should be unique and stable. If the order of the items changes, React will think it's a different object and will re-render it, which may lead to bugs and performance issues. You can use the item itself as a key in this case because it's guaranteed to be unique.